### ISSUE TITLE
Boolean schema is incorrectly typed as number

### ISSUE THREAD

**Issue by nkuek** [FormidableLabs/groqd#36]
```javascript
// schemas.ts

export const schemas = {
  string: z.string,
  number: z.number,
  boolean: z.number /* --> should be z.boolean */,
  unknown: z.unknown,
  null: z.null,
  undefined: z.undefined,
  date: dateSchema,
  literal: z.literal,
  union: z.union,
  array: z.array,
  object: z.object,
};
```

This leads to always triggering the invalid type error when using `q.boolean()`

**Comment by gksander** [comment]
GitHub Copilot has failed me! Thank you so much for catching this, we'll get the PR merged in and released ASAP!

**Comment by gksander** [comment]
Okay, released via 0.5.2!

**Comment by nkuek** [comment]
Thank you!
